aiogram/filters/callback_data.py

```diff
@@ -131,7 +132,7 @@ def unpack(cls: Type[T], value: str) -> T:
         for k, v in zip(names, parts):  # type: str, Optional[str]
             if field := cls.model_fields.get(k):
                 if v == "" and _check_field_is_nullable(field):
-                    v = None
+                    v = field.default if field.default is not PydanticUndefined else None
```

Comment:
In this situation, we are already in this state:
```python
v == field.default == "" 
```
You shouldn't set an empty string again. 
Just add another condition under which setting `v = None` will not happen.